src/robot/running/namespace.py

```diff
@@ -338,6 +339,42 @@ def _get_runner_from_libraries(self, name):
             return found[0]
         self._raise_multiple_keywords_found(name, found)
 
+    def _handle_private_runners(self, runners):
+        sum_public = sum("robot:private" not in runner.tags for runner in runners)
+        sum_private = sum("robot:private" in runner.tags for runner in runners)
```

Comment:
It's a bit questionable to use `sum()` with Boolean values but I guess it's ok here. The latter line could be simplified to:
```python
sum_private = len(runners) - sum_public
```